stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -403,12 +405,10 @@ public class MessageListViewModel(
             return
         }
 
-        if (message.id == lastSeenMessage.id) return
```

Comment:
The reason why we have this condition here is that we don't want to fetch/allocate any variables in case the IDs are the same. By doing it later, we fetch the 2 properties and potentially create 2 date instances, without having to do so.

To me this is not an optimization it's just making this more confusing and less optimized